app/views/items/index.html.erb

```diff
@@ -5,5 +5,5 @@
   <%= item.price %>
   <%= item.description %>
   <%= button_to "Add to Cart",
```

Comment:
Take a look at the HTML that this produces (view source on your dev site when you visit this page). The result is a button. Check the method (the HTTP verb that is going to be used when you click the button). When you hit this button, you're hitting your `create` method. Is that what you want to be hitting? My recommendation is to change to a link or figure out how to change the verb on a button. Let me know if you want to discuss further.